`krkn-lib` is incompatible with `kubernetes` Python client >= 36.0.0. All `ApiClient.call_api()` invocations that pass `response_type="str"` fail with:

This is because `kubernetes==36.0.0` (released May 20, 2026) renamed the `response_type` parameter to `response_types_map` as part of upgrading to OpenAPI Generator v6.6.0.

Since krkn-lib declares `kubernetes>=34.1.0` with no upper bound, any fresh installation resolves to kubernetes 36.x and is broken.

## Impact

- **7 call sites affected** across `krkn_kubernetes.py` (lines 3199, 3382, 3694) and `krkn_openshift.py` (lines 78, 124, 160, 546)
- Breaks: SA token creation, service patching, kubelet proxy metrics, platform detection, prometheus route discovery
- Any user doing `pip install krkn-lib` in a fresh environment hits this immediately
- Reported downstream: https://github.com/krkn-chaos/krkn/issues/1441

## Reproduction

```bash
pip install krkn-lib==6.1.0 kubernetes==36.0.2
python -c "
from kubernetes import config
from krkn_lib.k8s import KrknKubernetes
config.load_kube_config()
k = KrknKubernetes(kubeconfig_path='~/.kube/config')
print(k.create_token_for_sa('default', 'default'))
"

### Output:

``` [ERROR] failed to create token for SA: default on namespace: default with error: ApiClient.call_api() got an unexpected keyword argument 'response_type' None
```

## Proposed Fix
Pin the kubernetes dependency upper bound in pyproject.toml:
```
 kubernetes = ">=35.0.0,<36.0.0"
```
This aligns with what the krkn repo already enforces in its own requirements.txt and prevents pip from resolving to the incompatible 36.x series.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -10,7 +10,7 @@ homepage = "https://github.com/redhat-chaos/krkn"
 
 [tool.poetry.dependencies]
 python = "^3.11"
-kubernetes = ">=34.1.0"
+kubernetes = ">=35.0.0,<36.0.0"
 sphinxnotes-markdown-builder="^0.5.6"
 requests = ">=2.32.4"  # Fix CVE GHSA-9hjg-9r4m-mvj7, GHSA-9wx4-h78v-vm56, GHSA-gc5v-m9x4-r6x2
 kubeconfig = "^1.1.1"

